clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/ActiveActiveStoreIngestionTask.java

```diff
@@ -957,12 +964,6 @@ protected void startConsumingAsLeader(PartitionConsumptionState partitionConsump
       LOGGER.warn(
           "Failed to reach broker urls {}, will schedule retry to compute upstream offset and resubscribe!",
           unreachableBrokerList.toString());
-      // We won't attempt to resubscribe for brokers we couldn't compute an upstream offset accurately for. We'll
-      // reattempt subscription later
-      // Queue up repair here:
```

Comment:
Nitpick: could you please modify this comment a bit and move to the above exception branch? So that readers know that by not updating the map, subscription will not happen for the unreachable brokers?